Files/View Models/SettingsViewModel.cs

```diff
@@ -300,7 +300,33 @@ private async void DetectOneDrivePreference()
                 PinOneDriveToSideBar = false;
             }
         }
+        public bool OpenPropertiesInMultipleWindows
+        {
+            get => (bool)GetSettingsValue("OpenPropertiesInMultipleWindows",false);
+            set => localSettings.Values["OpenPropertiesInMultipleWindows"] = (bool)value;
+        }
+        public bool ShowFileOwner
+        {
+            get => (bool)GetSettingsValue("ShowFileOwner", false);
+            set => localSettings.Values["ShowFileOwner"] = (bool)value;
+        }
 
+        protected TResult GetSettingsValue<TResult>(string name, TResult defaultValue)
```

Comment:
Is there a reason you built your own GetSettingsValue when one is already implemented in the SettingsViewModel?

Reply:
Resolved in this [commit](https://github.com/files-community/files-uwp/pull/1055/commits/adaff7ea13068ec6aa50d07d255b341c2b3e132f)